### PR TITLE
Login view

### PR DESCRIPTION
**Pull request by NadNadou**
Hi, 

I've updated the Login view. This is the final view :

Login_final view

### CODE REVIEW

File: client/src/views/login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                container
+                direction="column"
+                justify="center"
+                className={classes.input}
+              >
+                <TextField
+                  placeholder="Email"
+                  ></TextField>
+
+                <TextField
```
Comment (notAnkur) [inline client/src/views/login/Login.jsx:66]: type="password"

File: client/src/views/login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+              </Grid>
+
+              <Grid
+                container
+                direction="column"
+                justify="center"
+                className={classes.input}
+              >
+                <TextField
```
Comment (notAnkur) [inline client/src/views/login/Login.jsx:61]: type="email"

**Comment by NadNadou** [comment]
Type modification for password and email

**Comment by NadNadou** [comment]
Hi Anantakur, 

I have an issue with pull request from my repo to ztm repo. Indeed, when I compare my repo to the Ztm I don't have the option to "create a pull request". Could you help me please ? 

image

**Comment by notAnkur** [comment]
Since a PR was already opened by you, the new commit was added to that pr

**Pull request merged**